akka-docs/src/main/paradox/stream/operators/ActorFlow/ask.md

```diff
@@ -52,11 +51,11 @@ Java
 
 **emits** when the futures (in submission order) created by the ask pattern internally are completed
 
-**backpressures** when the number of futures reaches the configured parallelism and the downstream backpressures
+**backpressure** when the number of futures reaches the configured parallelism and the downstream backpressure
 
 **completes** when upstream completes and all futures have been completed and all elements have been emitted
 
-**fails** when the passed in actor terminates, or a timeout is exceeded in any of the asks performed
+**fails** when the passed in an actor terminates, or a timeout exceeds in any of the asks performed
```

Comment:
'the passed in actor' is the subject of this sentence, inserting 'an' here makes it invalid.

While changing the passive 'is exceeded' into the active 'exceeds' is probably nicer, the sentence needs to be rearranged further. Perhaps:

```suggestion
**fails** when the passed-in actor terminates, or when any of the `ask`s exceed a timeout
```